.github/workflows/linux_cuda.yml

```diff
@@ -0,0 +1,204 @@
+name: Linux CUDA Plugin (Ubuntu 20.04, Python 3.11)
+on:
+  workflow_dispatch:
+  schedule:
+    # run daily at 00:00
+    - cron: '0 0 * * *'
+#  pull_request:
+#    paths-ignore:
+#      - '**/docs/**'
+#      - 'docs/**'
+#      - '**/**.md'
+#      - '**.md'
+#      - '**/layer_tests_summary/**'
+#      - '**/conformance/**'
+#  push:
+#    paths-ignore:
+#      - '**/docs/**'
+#      - 'docs/**'
+#      - '**/**.md'
+#      - '**.md'
+#      - '**/layer_tests_summary/**'
+#      - '**/conformance/**'
+#    branches:
+#      - master
+
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}-linux-cuda
+  cancel-in-progress: true
+
+jobs:
+  Build:
+    # TODO: remove. Temporary measure to prevent the workflow from scheduling on forks.
+    if: ${{ github.repository_owner == 'openvinotoolkit' }}
+    defaults:
+      run:
+        shell: bash
+    runs-on: ubuntu-20.04-8-cores
+    container:
+      image: nvidia/cuda:11.8.0-runtime-ubuntu20.04
+    env:
+      CMAKE_BUILD_TYPE: 'Release'
+      CMAKE_GENERATOR: 'Ninja'
+      CMAKE_CXX_COMPILER_LAUNCHER: ccache
+      CMAKE_C_COMPILER_LAUNCHER: ccache
+      CMAKE_CXX_LINKER_LAUNCHER: ccache
+      CMAKE_C_LINKER_LAUNCHER: ccache
+      BUILD_TYPE: 'Release'
+      OPENVINO_REPO: ${{ github.workspace }}/openvino
+      OPENVINO_CONTRIB_REPO: ${{ github.workspace }}/openvino_contrib
+      BUILD_DIR: ${{ github.workspace }}/build
+      INSTALL_DIR: ${{ github.workspace }}/install_pkg
+      DATA_PATH: ${{ github.workspace }}/testdata
+      MODELS_PATH: ${{ github.workspace }}/testdata
+      DEBIAN_FRONTEND: 'noninteractive'
+    steps:
+
+      - name: Install Prerequisites
+        run: |
+          apt update
+          apt install -y git curl git git-lfs unzip wget
```

Comment:
we already install build dependencies below install_build_dependencies.sh
Do we need this 'custom' step?

Reply:
Yes as we use `nvidia/cuda:11.8.0-runtime-ubuntu20.04` and it lacks these tools. Without them, the checkout is impossible.